paasta_tools/cli/utils.py

```diff
@@ -305,7 +305,7 @@ def guess_instance(service, cluster, args):
     else:
         try:
             instances = list_all_instances_for_service(
-                service=service, cluster=cluster, instance_type=None, soa_dir=args.yelpsoa_config_root)
```

Comment:
my guess is that ``local_run`` probably cares about this, so I'd be surprised if you can remove this without any side effects.

If you are going to remove it, then please remove the cluster parameter from the function definition, given it's not used anywhere else.